server/app.js

```diff
@@ -1,21 +1,25 @@
-const express = require("express");
-const morgan = require("morgan");
-const cors = require("cors");
-const bodyParser = require("body-parser");
-const mongoose = require("mongoose");
+const express = require("express")
+const morgan = require("morgan")
+const cors = require("cors")
+const bodyParser = require("body-parser")
+const app = express()
+// const mongoDb = require('./config/mongoDB') 
 
-const app = express();
+//parsers
+app.use(cors())
+app.use(morgan("dev"))
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
 
-const routes = require("./api/routes/");
 
-mongoose.connect("mongodb://localhost:27017/notes-app");
-mongoose.Promise = global.Promise;
+// redirector
+app.get('*', (req, res) => {
+  res.status(404)
+})
 
-app.use(cors());
-app.use(morgan("dev"));
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
 
-app.use("/", routes);
+//routes
+app.use("/", require("./routes/notes"))
```

Comment:
Uh oh! The above `*` route will actually be executed first where we are setting status as 404.
Move the wildcard route handler after this.

Reply:
done